source/SpinalHDL/Structuring/components_hierarchy.rst

```diff
@@ -58,37 +58,34 @@ The syntax to define inputs and outputs is the following:
      - Create an input/output of the corresponding type
      - Bits/UInt/SInt
    * - in/out(T)
-     - | For all other data types, you may have to add some brackets
-       | around it. Sorry this is a Scala limitation.
+     - For all other data types, you may have to add some brackets around it. Sorry, this is a Scala limitation.
      - T
    * - master/slave(T)
-     - | This syntax is provided by the `spinal.lib` (If you call your object `slave`,
-       | then call `spinal.lib.slave` instead). T should extend IMasterSlave –
-       | Some documentation is available :ref:`here <interface_eaxample_apb>`.
-       | You may not actually need the brackets, so `master T` is fine as well.
+     - This syntax is provided by the ``spinal.lib`` library (If you annotate your object with the ``slave`` syntax, then import ``spinal.lib.slave`` instead).
+       T should extend ``IMasterSlave`` – Some documentation is available :ref:`here <interface_eaxample_apb>`. You may not actually need the brackets, so ``master T`` is fine as well.
      - T
 
 
 There are some rules to follow with component interconnection:
 
 
-* Components can only read output and input signals of child components
-* Components can read their own output port values (unlike VHDL)
+* Components can only `read` output and input signals of child components.
```

Comment:
As below, i personally prefer inline formatting for reference code spinets. For putting emphasis on words I would rater user the usual **bold** and _Italic_ style.

```suggestion
* Components can only **read** output and input signals of child components.
```

Reply:
I think in rst, single backticks \` cause *italicized* formatting, but bolds are definitely fine with me for this use case. I wanted to call some attention to that part of the sentence, because it was easy to miss when I was first skimming that page of the docs.